src/main/java/org/web3j/console/services/Updater.java

```diff
@@ -15,74 +15,60 @@
 import java.io.IOException;
 
 import com.github.zafarkhaja.semver.Version;
-import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 import okhttp3.*;
 
 import org.web3j.console.utils.CliVersion;
-import org.web3j.console.utils.OSUtils;
 
 import static org.web3j.console.config.ConfigManager.config;
 
 public class Updater {
-    private static final String DEFAULT_UPDATE_URL =
-            "https://internal.services.web3labs.com/api/epirus/versions/latest";
+    static String GITHUB_API_URL =
```

Comment:
Declaring values like this it's unsafe and as your are doing it for the test purpose you are breaking the Dependency Inversion code principle. Best here is to send this value where is used as parameter to the method and declare is as constant in upper modules.

Reply:
Updated the test and switched back to private static final